piker/cli/__init__.py

```diff
@@ -108,6 +115,27 @@ async def main():
                     f'config: {pformat(config)}'
                 )
 
+            if es:
```

Comment:
Looking good, but we need to move the database initialization code inside the `open_pikerd` context manager, to make it more generic and not duplicate code on tests.

Reply:
Ok, working on it